mrp_production_add_middle_stuff/README.rst

```diff
@@ -0,0 +1,5 @@
+
+MRP Production add middle stuff
```

Comment:
Main problem I have with this module is the naming
mrp_production_add_raw_material?